genesis/src/lib.rs

```diff
@@ -92,13 +100,21 @@ impl GenesisNetworkTrait for GenesisNetwork {
     }
 }
 
-/// [`RawGenesisBlock`] is an initial block of the network
-#[derive(Debug, Clone, Default, Deserialize, Serialize, IntoSchema)]
-#[serde(transparent)]
+/// Mock of [`Validator`](iroha_data_model::permission::Validator) for unit tests
+///
+/// Aliased to `Validator` when `cfg(test)`.
+#[cfg(test)]
+#[derive(Debug, Clone, Copy, Deserialize, Serialize, IntoSchema)]
 #[repr(transparent)]
+pub struct MockValidator;
```

Comment:
Why do you have `MockValidator` defined multiple times? here and in the `core` module

Reply:
1. It's different crates
2. This mocks serve to a two different puproses. In `genesis` it's just a stub that does nothing. In `core` it has some mocking methods

Reply:
related to [this comment](https://github.com/hyperledger/iroha/pull/3329#discussion_r1162651751)